app/src/androidTest/java/com/hedvig/app/feature/embark/marketselected/MarketSelectedScreen.kt

```diff
@@ -0,0 +1,12 @@
+package com.hedvig.app.feature.embark.marketselected
+
+import com.agoda.kakao.screen.Screen
+import com.agoda.kakao.text.KButton
+import com.agoda.kakao.text.KTextView
+import com.hedvig.app.R
+
+class MarketSelectedScreen: Screen<MarketSelectedScreen>() {
```

Comment:
```suggestion
object MarketSelectedScreen: KScreen<MarketSelectedScreen>() {
```

Also, don't forget to format your files 😉